signer/migrations/0004__create_functions.sql

```diff
@@ -0,0 +1,50 @@
+-- TVF for fetching a Bitcoin blockchain from a given block hash, only looking
```

Comment:
Can we write out TVF as table-valued function. It's not obvious what this means (I had to look it up).

Reply:
Yeah I can update that

Reply:
Updated in [e7b7b8f](https://github.com/stacks-network/sbtc/pull/710/commits/e7b7b8f6464797271ddbed772b81354247214f08)